aiohttp/server.py

```diff
@@ -90,8 +89,7 @@ def __init__(self, *, loop=None,
                  keep_alive_on=True,
                  timeout=0,
                  logger=server_logger,
-                 access_log=None,
```

Comment:
It works even slower then current implementation if AccessLogger created in the constructor.

Converting from %h -> %(h)s takes too much time.

I have one idea have to leave current signatures and make it work faster, but then access_log_format will be not apache compatible. It would be something like:

...
'%(a)s' -- remote IP-address
'%(i_Foobar)s' -- 'Foobar' header line of input headers
'%(e_FOOBAR)s' -- The contents of the environment variable FOOBAR
...
Is it ok?